src/PostgREST/Error.hs

```diff
@@ -289,6 +291,9 @@ instance JSON.ToJSON Error where
   toJSON (JwtTokenInvalid message) = JSON.object [
     "message" .= (message :: Text)]
   toJSON NotFound = JSON.object []
+  toJSON RpcNotFound = JSON.object [
+    "hint"    .= ("No function matches the given name and argument types. If a new function was created in the database with this name, try updating the schema cache." :: Text),
+    "message" .= ("This function does not exist" :: Text)]
```

Comment:
To be more precise, the error should be phrased in terms of `Could not find..`:

```haskell
    "message" .= ("Could not find the add_them(a,b) function")
```

Reply:
Agree, it's a better description of the issue.